cmd/utils.go

```diff
@@ -95,9 +95,55 @@ func splitStr(path, sep string, n int) []string {
 	return splits
 }
 
+func buildS3ConfigFromEnv(alias, urlStr, envConfig string) (*Config, *probe.Error) {
+	// We have a valid alias in env. We populate the
+	// credentials from env.
+	s3Config := new(Config)
+
+	// Fetch keys from the environement, otherwise, get them from the config file
+	keys := splitStr(envConfig, ":", 4)
+	var (
+		accessKey     = keys[0]
+		secretKey     = keys[1]
+		hostURL       = keys[2]
```

Comment:
Isn't hostURL == urlStr in this case ?

Reply:
Not always @vadmeste we are overriding it.